app/models.py

```diff
@@ -103,12 +104,16 @@ def query_in_deployment(cls):
         return cls.query.filter(cls.deployment.in_(current_app.config['SEARCH_DEPLOYMENTS']))
 
     @property
+    def full_name(self):
+        return u"%s %s" % (self.first_name, self.last_name)
+
+    @property
     def display_in_search(self):
         '''
         Determine whether user has filled out bare minimum to display in search
         results.
         '''
-        return self.first_name is not None and self.last_name is not None
+        return bool(self.first_name and self.last_name)
```

Comment:
This is more accurate -- not a huge fan of the syntax, so maybe update the docstring to say what this does (makes sure that first name last name are both non-null and non-blank)?